emu/src/cpu/arm7tdmi.rs

```diff
@@ -308,6 +308,28 @@ impl Arm7tdmi {
         Some(SIZE_OF_THUMB_INSTRUCTION)
     }
 
+    fn load_store_immediate_offset(&mut self, op_code: ThumbModeOpcode) -> Option<u32> {
+        let byte_word: ReadWriteKind = op_code.get_bit(12).into();
+        let load_store: LoadStoreKind = op_code.get_bit(11).into();
+        let offset5 = op_code.get_bits(6..=10) as u32;
+        let rb = op_code.get_bits(3..=5);
+        let rd = op_code.get_bits(0..=2);
+
+        let base = self.registers.register_at(rb.try_into().unwrap());
+        let value = base.wrapping_add(offset5);
```

Comment:
I think `address` is better as name here to be consistent with other functions

Reply:
Yes!